sk22/megalodon
# feat: allow playing multiple media

Closes https://github.com/sk22/megalodon/issues/733.

Adds a new setting which allows the user to view media in the app, while continuing to listen to other audios, for example listening to music. 

![Overlay Media setting](https://github.com/sk22/megalodon/assets/63370021/4c80d3e8-3100-4670-aa2a-e0febe0c3cfe)

### Testing
1. Play a sound in the background
2. Play a video with sound in the app
3. Notice how the music stops
4. Toggle the setting and play the video again
5. Notice how the music continues in the background
